libbeat/processors/communityid/config.go

```diff
@@ -0,0 +1,48 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package communityid
+
+type config struct {
+	Fields fieldsConfig `config:"fields"`
+	Target string       `config:"target"`
```

Comment:
I don't know how widespread is the use of the hash `seed` parameter in Community-ID, maybe it's worth to add a config option to set it.

Reply:
I added the parameter. I didn't realize that was a thing.